linters/detect_secrets/detect_secrets_annotations.js

```diff
@@ -0,0 +1,30 @@
+// Copyright 2019 VMware, Inc.
+// SPDX-License-Identifier: BSD-2-Clause
+
+const { getAnnotationLevel } = require('../../annotations_levels')
+
+/**
+ * @param {*} issue an issue from which the annotation will be build
+ * @param {String} filePath file path where the issue is located
+ * @return {Object} returns an annotation object as specified here:
+ * https://developer.github.com/v3/checks/runs/#annotations-object
+ */
+function getAnnotation (issue, filePath) {
+  const path = filePath
+  const startLine = issue.line_number
+  const endLine = startLine
+  const annotationLevel = getAnnotationLevel('HIGH', 'LOW')
```

Comment:
Should the severity always be high and confidence always low for these results?

Reply:
That's something we should discuss in our issue tracker
https://github.com/vmware/precaution/issues/209
I just put the severity and confidence Bandit uses when reporting hardcoded credentials.

Reply:
You might want to look at https://github.com/lyft/bandit-high-entropy-string. This is a bandit plugin to do detect-secrets like testing.  As part of the plugin, it defines severity and confidence.